src/middleware/company.authentication.middleware.ts

```diff
@@ -48,3 +34,26 @@ export const companyAuthentication = async (req: Request, res: Response, next: N
     next(err);
   }
 };
+
+async function processTransaction (entityNumber: string, returnURL: string, req: Request): Promise<[string, string]> {
+  if (req.path.endsWith(`/${RESUME}`)) {
```

Comment:
This if should really be moved outside this function. 

if ((req.path.endsWith(`/${RESUME}`)) {
   [entityNumber, returnURL] = await processTransaction(entityNumber, returnURL, req);
}